services/director/src/simcore_service_director/producer.py

```diff
@@ -1149,50 +1139,48 @@ async def get_service_details(app: FastAPI, node_uuid: str) -> dict:
 
 
 @retry(
-    wait=wait_fixed(2),
+    wait=wait_random_exponential(min=1, max=5),
```

Comment:
Question: what happens if a services does not expose the save state? Let's say the API endpoint is missing.

Reply:
this did not change as far as I know

Reply:
see https://www.python-httpx.org/quickstart/#exceptions
```
The RequestError class is a superclass that encompasses any exception that occurs while issuing an HTTP request
```

Reply:
I see that we filter some errors.